### ISSUE TITLE
Refactor flash() as a controller helper

### ISSUE THREAD

**Issue by pacovell** [donedotcom/dispatch#3]
We should have a standard way to add helpers like flash() and render() to the controller context.  Perhaps these come in a collection of before hooks that can be easily configured or added to?

**Comment by joshleitzel** [comment]
Paul,

I'm confused what you mean by this. Do these 'built-in' before hooks (e.g., render helper, flash) live in the Controller class, or inside an instance of a controller?

I'm envisioning something like this:

```
 this.before('all', function () {
      return this.renderHelper.render(req, res, res.next, options || {});
 });
```

But it's unclear to me where that would go or how different it is from what's already there.

**Comment by pacovell** [comment]
Closed by commit 56befce8fdc218eb93c19b4458a90bd3a39fef02
